server.js

```diff
@@ -0,0 +1,15 @@
+const express = require("express")
+const app = express()
+const port = 3000
+
+app.get('/ping',(req,res)=>{
+    res.send("pong")
+})
+
+app.get('/',(req,res)=>{
```

Comment:
Consider adding a catch-all route at the end of all your routes to handle 404 errors. [medium]